### PR TITLE
Add harvest log API usage documentation

### PR DESCRIPTION
**Pull request by polarp**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -134,6 +134,37 @@ Database logger configuration(optional)
...
+
+    $ curl {ckan_url}/api/3/action/harvest_log_list
+
+Replace {ckan_url} with the url from your CKAN instance.
+
+Allowed parameters are: 
+
+    * level (filter log records by level)
+
+    * per_page (used for pagination)
```
Comment (amercader) [inline README.rst]: The convention for paginated endpoints in CKAN is to use `limit`. Can you change this now the the API is quite recent? You can keep support for `per_page` but undocumented. Also it can go on a separate PR if you want

Comment (polarp) [inline README.rst]: @amercader I updated the pagination limit param name and updated the readme.

You can checkout the new PR [here](https://github.com/ckan/ckanext-harvest/pull/247).

**Comment by amercader** [comment]
Closed in favour of #247
